### ISSUE TITLE
Extend seed_genres in recommendations request

### ISSUE THREAD

**Issue by ClaudioAmato** [spotify/web-api#1526]
Hi,

I would like to make a **_request_ about getting recommendation with genres** (I mean this: https://developer.spotify.com/documentation/web-api/reference/browse/get-recommendations/):

Why don't some genres like "**_Rap_**" or "**_Trap_**" (just for example) exist? I know that we have to follow the genres here listed (https://developer.spotify.com/console/get-available-genre-seeds/) but some of these genres are famous a lot and this is not possible to leave them off (and "_rap or trap_" are only an example).

Why can't we find for more genres? Actually we can use 126 genres seeds but as you can see here (http://everynoise.com/everynoise1d.cgi?scope=all&vector=popularity) there are about 4000 genres and the "_mainstream only_" are about 260.

Regards.

**Comment by bpingris** [comment]
Yeah indeed I believe this endpoint is not updated, if you try to gather genres of musics by using the artist endpoint, you can have 250+- different genres
